src/version.c

```diff
@@ -96,13 +96,12 @@ int get_current_version(char *path_prefix)
 	FILE *file;
 	int v = -1;
 	char *buildstamp;
-	char *src, *dest;
 
-	string_or_die(&buildstamp, "%s/usr/lib/os-release", path_prefix);
```

Comment:
Is there a reason we even have a version file then? They are always in sync with one another, and it's easier to read this simple file vs parsing the os-release file in C.